functions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+function getDb() : PDO {
+    $db = new PDO('mysql:host=db; dbname=damian-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function retrieveAlbums(PDO $db) : array {
+    $query = $db->prepare("SELECT `name`, `artist`, `release-year`, `genre`, `record-label`, `image-link` FROM `albums`");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+function displayAlbums(array $albums) : string {
+
+    if($albums === []) {
+        $displayalbums = 'ERROR ERROR ERROR';
+    } else {
```

Comment:
the code in the else doesnt need to be in an else. can just make the if statement have no else and pop the foreach code below. it will only be accessed if the if statement isnt true

Reply:
done